### DISCUSSION TITLE
The page inside the index folder covers the routing of the same level

### DISCUSSION THREAD

**Discussion by tbontb-iaq** [posva/unplugin-vue-router#294] [Questions and Help] [answered]
When I have the following file structure:

```text
src/pages
├── home.vue
├── index
│   └── home.vue
└── index.vue
```

And access path `/home`, it renders the `index.vue` and nested `index/home.vue`.

How do I access `home.vue`?

Can these two situations exist at the same time and have different URLs?

**Comment by posva** [comment]
The issue with this structure is that `home.vue` and `index/home.vue` produce the same path (`/home`). In terms of routing, that is not a good approach because directly visiting `/home` ends up on one of them and the other route is always ignore

**Reply by tbontb-iaq** [reply]
Can they be generated as different paths? Such as `/index/home` and `/home`?

**Reply by posva** [reply, marked as answer]
_index_ is a reserved name but you can use `definePage()` to replace the `path` property with whatever you want. If you can just avoid a route with _index_ in it, that will be better overall
